specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-05-30-preview/share.json

```diff
@@ -0,0 +1,2212 @@
+          "type": "string"
+        },
+        "senderEmail": {
+          "description": "Email address of the sender.",
+          "type": "string",
+          "readOnly": true
+        },
+        "senderName": {
+          "description": "Name of the sender",
+          "type": "string",
+          "readOnly": true
+        },
+        "senderTenantName": {
+          "description": "Tenant name of the sender",
+          "type": "string",
+          "readOnly": true
+        },
+        "sentAt": {
+          "format": "date-time",
+          "description": "Gets the time at which the invitation was sent. Represented in the standard date-time format as defined by [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339)",
+          "type": "string",
+          "readOnly": true
+        },
+        "shareStatus": {
+          "$ref": "#/definitions/ShareStatus"
+        },
+        "state": {
+          "$ref": "#/definitions/State",
+          "readOnly": true
+        },
+        "targetActiveDirectoryId": {
+          "format": "uuid",
+          "description": "The target azure active directory id the invitation is sent to.",
+          "type": "string"
+        },
+        "targetObjectId": {
+          "format": "uuid",
+          "description": "The target object id in the azure active directory the invitation is sent to.",
+          "type": "string"
+        }
+      }
+    },
+    "ShareCounts": {
+      "description": "A ShareCounts Data Transfer Object.",
+      "type": "object",
+      "properties": {
+        "receivedSharesCount": {
+          "format": "int32",
+          "description": "A count of Received Shares associated with the Microsoft.Azure.Purview.Share.ApiService.V3.DataTransferObjects.ShareResource.",
+          "type": "integer"
+        },
+        "sentSharesCount": {
+          "format": "int32",
+          "description": "A count of Sent Shares associated with the Microsoft.Azure.Purview.Share.ApiService.V3.DataTransferObjects.ShareResource.",
+          "type": "integer"
+        }
+      }
+    },
+    "ShareKind": {
+      "description": "Defines the supported types for share.",
+      "enum": [
+        "InPlace"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ShareKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "InPlace",
+            "description": "The type of share is in-place sharing",
+            "name": "In place share kind"
+          }
+        ]
+      }
+    },
+    "ShareResource": {
+      "description": "A share resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "shareCounts": {
+          "$ref": "#/definitions/ShareCounts"
+        },
```

Comment:
Why nest these down inside a "ShareCounts" object? I think it will be better for users to simply put the two properties from "ShareCounts" here at the top level.